gk/main.c

```diff
@@ -745,6 +822,22 @@ gk_stage2(void *arg)
 	if (ret < 0)
 		goto cleanup;
 
+	if (ipv6_if_configured(&gk_conf->net->front)) {
+		int i;
+		for (i = 0; i < gk_conf->num_lcores; i++) {
+			unsigned int lcore = gk_conf->lcores[i];
+			ret = register_ipv6_acl(&gk_conf->net->front,
```

Comment:
The LLS block should register the ACLs it needs, not the helping blocks. The same goes for the GT block.